src/pages/home/report/ReactionList/PopoverReactionList/index.js

```diff
@@ -0,0 +1,33 @@
+import React, {forwardRef, useImperativeHandle, useRef, useState} from 'react';
+import BasePopoverReactionList from './BasePopoverReactionList';
+
+const PopoverReactionList = forwardRef((props, ref) => {
+    const innerReactionListRef = useRef();
+    const [reactionListReportActionID, setReactionListReportActionID] = useState('');
+
+    /**
+     * Show the ReactionList modal popover.
+     *
+     * @param {Object} [event] - A press event.
+     * @param {Element} reactionListAnchor - reactionListAnchor
+     * @param {String} emojiName - Name of emoji
+     * @param {String} reportActionID
+     */
+    const showReactionList = (event, reactionListAnchor, emojiName, reportActionID) => {
+        setReactionListReportActionID(reportActionID);
+        innerReactionListRef.current.showReactionList(event, reactionListAnchor, emojiName);
```

Comment:
Shouldn't we wait for the set state to complete before calling `showReactionList`?